examples/scripts/denoise_cplx_tv_pdhg.py

```diff
@@ -5,8 +5,8 @@
 # with the package.
 
 r"""
-Complex Total Variation Denoising
-=================================
+Complex Total Variation Denoising with PDHG Solver
```

Comment:
If easy, reference the other similar example (with NLPADMM) from here.

Reply:
Not so easy. There are some other places in which such cross-references would also be useful. Will leave as a todo for now.